.github/workflows/node.js.yml

```diff
@@ -29,12 +29,6 @@ jobs:
       # Install dependencies
       - run: npm ci
 
-      # Install s2i
-      - run: |
-          wget https://github.com/openshift/source-to-image/releases/download/v1.3.1/source-to-image-v1.3.1-a5a77147-linux-amd64.tar.gz -P ~
-          tar -xvf ~/source-to-image-v1.3.1-a5a77147-linux-amd64.tar.gz -C $HOME/
-          echo "$HOME/" >> $GITHUB_PATH
-
       # Linting
       - run: npm run check-style
       - run: npm run lint
```

Comment:
You no longer need to run `npm run build` here, as it's built inside of docker-compose.